src/components/collapsible-motion/collapsible-motion.js

```diff
@@ -1,27 +1,9 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 import invariant from 'tiny-invariant';
-import stringHash from '@sindresorhus/string-hash';
-import styleInject from 'style-inject';
+import { keyframes } from 'emotion';
```

Comment:
Can we add this as a proper dependency now that we depend on it top-level so we don't just transiently live by it through react-select?

Reply:
can add as regular dep and add it as an external to rollup

Reply:
It has been added as a `peerDependency` (and as a `devDependency`). `react-select` also depends on `emotion` and is a `peerDependency` itself.

Reply:
Ok.